src/components/results/common/computation-report-viewer.tsx

```diff
@@ -50,6 +52,11 @@ export const ComputationReportViewer: FunctionComponent<
 
     useEffect(() => {
         if (studyUuid && currentNode?.id) {
+            // use a timout to avoid having a loader in case of fast promise return (avoid blink)
```

Comment:
Use `useOpenShortWaitFetching` ?

Reply:
I dont see how this component can help here.